index.js

```diff
@@ -71,6 +72,14 @@ function bootApp() {
       .end()
   })
 
+  app.get('/health', (req, res) => {
+    if (client.ping()) {
```

Comment:
Can you maybe wrap the `ping` function in our `lib/cache` module rather than using the underlying client directly? I’m not even really sure why we export the `client`, maybe for testing purposes? If so, can you also leave a comment above the `client` export saying it should only be accessed for testing purposes?

Reply:
Right.  Will wrap that in a `cache.available` method or something...